src/module/VisualizationParser.ts

```diff
@@ -118,4 +114,11 @@ export class VisualizationParser {
       edges: Array.from(me.values()),
     };
   }
+
+  /**
+   * Return true if an item is a node
+   */
+  public static isNode(item: IItem): item is INode {
```

Comment:
If the rest client has linkurious-shared as dependency why not use ```Tools.isNode() ``` ?

Reply:
Actually is method is not used anymore :) I removed it